### DISCUSSION TITLE
Small area flow compensation per filament rather than per print profile

### DISCUSSION THREAD

**Discussion by mggevaer** [OrcaSlicer/OrcaSlicer#5693] [Ideas]
Small area flow compensation is a great feature, however, it's hard to find a one-size fits all calibration for multiple materials, since different filaments behave differently.
Therefore I believe it makes more sense to have the small area flow compensation model be part of the filament profile.
Any thoughts?

**Comment by markaudacity** [comment]
This this this! I have thought this for several different settings that are only in printer settings, not filament settings--elephant's foot compensation, ironing, and bridging parameters leap to mind. TPU behaves almost completely differently than PAHT-CF, for example, in ways that need addressing beyond flow rate, cooling, and retraction settings.

**Comment by gerolori** [comment]
I'll bump this up. Definetly needed
Even between PLA and PETG I have wildly different settings, mostly because they have very different max volumetric speed and pressure advance too. I'd love to be able to set it on the filament

**Comment by ItsReckliss** [comment]
Just learning about tuning small area flow but id imagine a gummier filament like petg would need a harsher curve than something like pla. This should be implemented.

**Comment by buttabean** [comment]
i just posted this in the discord then it dawned on me to search here lol "is there a reason ironing flow % isn't set in the material profile instead of the process? isn't it a calibration that is tied to material type? seems like it would be less hassle and organizable to have it setup similar to how there's a object overrides section but you could place that inside the material profile so it overrides the process settings"

the option to override setting specific to the filament would be ideal since the flow is tied to the material calibration and you have different rates for different nozzles too. so hope this gets added.

**Comment by gretel** [comment]
using this curve for `PETG-HF`:

```
0,0;
0.2,0.25;
0.4,0.45;
0.6,0.55;
0.8,0.65;
1.5,0.75;
2,0.82;
3,0.90;
5,0.96;
10,1;
```
